cypress/e2e/logout.cy.js

```diff
@@ -0,0 +1,24 @@
+// testing suite for logout component
+
+describe('Logout Functionality', () => {
+    beforeEach(() => {
+        // login using custom command found in /support/commands.js
+        cy.login('kingliver', 'cats');
+        // visits profile page where the logout button is present
+        cy.contains('Profile').click();
+        cy.url().should('include', '/profile');
+        cy.get('[data-test="profile-page"]').should('be.visible');
+    });
+
+    it('logs out the user and redirects to login', () => {
+        // Ensure the page is loaded correctly
+        cy.url().should('include', '/profile');
+
+        // renders the logout button
+        cy.get('[data-test="logout-btn-profile"]').should('be.visible');
+        cy.get('[data-test="logout-btn-profile"]').click();
+
+        cy.url().should('include', '/login');
+    })
```

Comment:
If you have any conditional rendering for your logout button (like says log out if there is a user but changes to log in when they log out rather than just switching components altogether) I could potentially see a use in also double checking that the log out button is no longer visible or that the log in one replaces it, if that makes sense.

Reply:
The log out button just lives on the profile page and then the user gets redirected to the login screen. From there, all the pages are protected unless a user logs in with valid credentials, so they would not be able to access it. However, I agree that an extra step could be added in verifying that the log-out button is no longer rendering on the webpage! 
Submitted changes with commit: [f76aafc](https://github.com/abbiehuynh/my-crochet-corner/pull/15/commits/f76aafcc815ba2f005d5dfc53a31f4a7a7591ec6)